test/solver/batch_bicgstab_kernels.cpp

```diff
@@ -71,6 +82,7 @@ class BatchBicgstab : public CommonTestFixture {
         solve_lambda;
     Settings solver_settings{};
     std::shared_ptr<solver_type::Factory> solver_factory;
+    std::shared_ptr<solver_type::Factory> precond_solver_factory;
```

Comment:
this is not used. Is a test missing?

Reply:
Just a code copy leftover. I moved the test with the solver to `test/preconditioner/...`